serverless/eventing/event-sources/serverless-connecting-brokers-sinks.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="serverless-connecting-brokers-sinks"]
+= Connecting brokers to sinks
+:context: serverless-connecting-brokers-sinks
+
+toc::[]
+
+Brokers can be used in combination with triggers to deliver events from an event source to an event sink. Events are sent from an event source to a broker as an HTTP `POST` request. After events have entered the broker, they can be filtered by https://github.com/cloudevents/spec/blob/v1.0/spec.md#context-attributes[CloudEvent attributes] using triggers, and sent as an HTTP `POST` request to an event sink.
```

Comment:
I think we're not supposed to include links to github from our docs, just FYI 🙂

Reply:
Removed the link from the content! Thanks.